src/assets/data/faq/index.js

```diff
@@ -105,7 +105,7 @@ const data = {
             question: "Does Meshery manage service meshes?",
             category: "Meshery",
             answer: ["Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes."],
-            link: "/projects/meshery",
+            link: "/meshery",
```

Comment:
Link should be `/service-mesh-management/meshery`

Reply:
It has redirects btw

Reply:
It is better to keep the real link in the site though